src/persist/src/nemesis/direct.rs

```diff
@@ -613,6 +616,37 @@ mod tests {
         nemesis::run(10, GeneratorConfig::default(), direct);
     }
 
+    impl StartRuntime for S3BlobConfig {
+        fn start_runtime(&mut self, unreliable: UnreliableHandle) -> Result<RuntimeClient, Error> {
+            let runtime = Arc::new(AsyncRuntime::new().expect("failed to create async runtime"));
+            let guard = runtime.enter();
+            let config = self.clone();
+            let blob = S3Blob::open_exclusive(config, ("reentrance0", "direct_file").into())?;
+            let blob = UnreliableBlob::from_handle(blob, unreliable);
+            drop(guard);
+            runtime::start(
+                RuntimeConfig::for_tests(),
+                ErrorLog,
+                blob,
+                mz_build_info::DUMMY_BUILD_INFO,
+                &MetricsRegistry::new(),
+                Some(runtime),
+            )
+        }
+    }
+
+    #[test]
+    fn direct_s3() {
+        let runtime = Arc::new(AsyncRuntime::new().expect("failed to create async runtime"));
```

Comment:
I've paged out the context here completely but its surprising to me that we create a new async runtime here and in `start_runtime` a few lines above. I'm sure its fine but it might be worth a comment?

Reply:
we could avoid the second one if we impl'd StartRuntime for `(Arc<AsyncRuntime>, S3BlobConfig)`. do you prefer that? I'm happy either way. will add a comment if we decide to keep it as is

Reply:
no real preference

Reply:
it was easy/clean enough to do the impl over the tuple, so went ahead and did that